src/components/Launch/LaunchForm/constants.ts

```diff
@@ -18,21 +12,21 @@ export const formStrings = {
     launchPlan: 'Launch Plan'
 };
 
-type RoleTypesKey = 'iamRole' | 'k8sServiceAccount';
-export const roleTypes: { [k in RoleTypesKey]: RoleType } = {
-    iamRole: {
-        helperText: 'example: arn:aws:iam::12345678:role/defaultrole',
-        inputLabel: 'role urn',
-        label: 'IAM Role',
-        value: 'assumableIamRole'
-    },
-    k8sServiceAccount: {
-        helperText: 'example: default-service-account',
-        inputLabel: 'service account name',
-        label: 'Kubernetes Service Account',
-        value: 'kubernetesServiceAccount'
-    }
-};
+// type RoleTypesKey = 'iamRole' | 'k8sServiceAccount';
+// export const roleTypes: { [k in RoleTypesKey]: RoleType } = {
+//     iamRole: {
+//         helperText: 'example: arn:aws:iam::12345678:role/defaultrole',
+//         inputLabel: 'role urn',
+//         label: 'IAM Role',
+//         value: 'assumableIamRole'
+//     },
+//     k8sServiceAccount: {
+//         helperText: 'example: default-service-account',
+//         inputLabel: 'service account name',
+//         label: 'Kubernetes Service Account',
+//         value: 'kubernetesServiceAccount'
+//     }
+// };
```

Comment:
Do we need this later?

Reply:
Whoops, meant to remove this; good catch - fixed.